front/pages/api/w/[wId]/mcp/index.ts

```diff
@@ -300,7 +298,7 @@ async function handler(
             status_code: 400,
             api_error: {
               type: "invalid_request_error",
-              message: "A server with this URL already exists",
+              message: "This internal tool has already been added",
```

Comment:
I assume there is no way to get the error through the UI, right? You'd need to call the API, but since it's an internal API, it can't actually happen?

It's fine to have the error handling, it's more me making sure I understand how this all fits together :)

Reply:
Got it.